core/test/resources/code-model.yaml

```diff
@@ -1,656 +1,421 @@
 {

```

Comment:
I think we should avoid formatting all YAML and JSON files under the `core/test/resources` path too